rust/template/differential_datalog/src/program/arrange.rs

```diff
@@ -18,10 +18,22 @@ use differential_dataflow::{
 use fnv::FnvHashMap;
 use std::ops::{Add, Mul, Neg};
 use timely::{
-    dataflow::scopes::{Child, Scope, ScopeParent},
+    dataflow::scopes::{Child, Scope},
     progress::{timestamp::Refines, Timestamp},
 };
 
+#[derive(Clone)]
+pub enum ArrangementFlavor<S, T>
+where
+    S: Scope,
+    S::Timestamp: Lattice + Refines<T>,
+    T: Lattice + Timestamp,
+{
+    Local(Arrangement<S, Weight, TValAgent<S::Timestamp>, TKeyAgent<S::Timestamp>>),
```

Comment:
Please add some comments, probably along the lines of "arrangement created in the local scope", "arrangement imported from the parent scope".